python/ray/train/_internal/state/schema.py

```diff
@@ -47,6 +47,55 @@ class TrainWorkerInfo(BaseModel):
     )
 
 
+@DeveloperAPI
+class MemoryInfo(BaseModel):
+    rss: int
+    vms: int
+    pfaults: Optional[int]
+    pageins: Optional[int]
+
+
+@DeveloperAPI
+class ProcessStats(BaseModel):
+    cpuPercent: float
+    # total memory, free memory, memory used ratio
+    mem: Optional[List[int]]
+    memoryInfo: MemoryInfo
```

Comment:
Why is `mem` a list and not individual fields? Why are these not part of `memoryInfo`?